model/seeker/getSeeker.js

```diff
@@ -1,7 +1,33 @@
 const client = require('../../database/pg.js');
 const utils = require('../../lib/hashUtils.js');
 
-getSeeker = (data, callback) => {
+getSeeker = (cookie, callback) => {
+  const sessionQueryString = `SELECT user_id FROM session WHERE cookie = $1`;
```

Comment:
Super small edit, when using just a regular string do '', instead of backticks, only do backticks for string literals